src/app/portlets/content-types/fields/content-type-fields-row/content-type-fields-row.component.ts

```diff
@@ -37,16 +45,21 @@ export class ContentTypeFieldsRowComponent extends BaseComponent {
      * @param field field to remove
      */
     onRemoveField(field: Field): void {
-        this.fieldRow.columns = this.fieldRow.columns.map(col => {
-            const index: number = col.fields.indexOf(field);
-
-            if (index !== -1) {
-                col.fields.splice(index, 1);
-             }
-            return col;
+        this.confirmationService.confirm({
+            accept: () => {
+                this.fieldRow.columns = this.fieldRow.columns.map(col => {
```

Comment:
Don't understand why this code is here... as far as I can see the accept should only emmit isn't? if not... please move this to an external method.